src/Application/Models/Mint.js

```diff
@@ -0,0 +1,44 @@
+import Denomination from './Denomination';
+
+const denominations = [
```

Comment:
really it would make more sense in some `constants/` file.  These things are comparably as static as, say, the array of quotes currently in `entrypoint.jsx` (which also needs more "this is a constant!" indication).  it's not a parameter to the constructor since it never changes -- the details (and order) of each denomination level will never change in this app, so no need to allow for them to